packages/pyright-internal/src/analyzer/typeUtils.ts

```diff
@@ -448,6 +448,15 @@ export interface MapSubtypesOptions {
     retainTypeAlias?: boolean;
 }
 
+export interface MapSubtypesOptions {
```

Comment:
This type already exists on line 436 just above here.

Reply:
oh. I thought I fixed those.

Reply:
ah, this is another merge conflict issues.